packages/tables/src/Table.php

```diff
@@ -339,6 +339,23 @@ public function getRecordUrl(Model $record): ?string
         return $callback($record);
     }
 
+    public function isRecordBulkActionEnabled(Model $record): bool {
```

Comment:
Please try to adhere to the existing code style.

Also please rename all methods and vars related to this to `isRecordSelectable`

```suggestion
    public function isRecordSelectable(Model $record): bool
    {
```

Reply:
This is not resolved. Please make sure it's called `isRecordSelectable` everywhere.

Reply:
Can you point me to the place where it is not changed? I cannot find it...

Reply:
Mark all old ocurrences. Should be easy to find via "Files" tab

Reply:
Also please test it to ensure it works after renaming ;)